src/services/lnd/onchain-service.ts

```diff
@@ -23,6 +23,8 @@ import { SECS_PER_5_MINS } from "@config"
 
 import { getActiveOnchainLnd } from "./utils"
 
+import { parseLndErrorDetails } from "."
```

Comment:
in general this can lead to a circular reference, maybe is better to move `parseLndErrorDetails` to utils

Reply:
https://github.com/GaloyMoney/galoy/pull/1382/commits/af811e72f1549544972e4dea4622a165a99f4dc8